optuna/integration/lightgbm_tuner/__init__.py

```diff
@@ -1,8 +1,15 @@
 from optuna._experimental import experimental
-from optuna.integration.lightgbm_tuner.sklearn import LGBMClassifier, LGBMModel, LGBMRegressor  # NOQA
-from optuna.integration.lightgbm_tuner.optimize import LightGBMTuner
 from optuna import type_checking
 
+try:
+    from optuna.integration.lightgbm_tuner.sklearn import LGBMClassifier, LGBMModel, LGBMRegressor  # NOQA
+    from optuna.integration.lightgbm_tuner.optimize import LightGBMTuner
```

Comment:
`_available = True` seems to be required here to pass `_check_lightgbm_availability()`.

Reply:
🙇